music21/metadata/__init__.py

```diff
@@ -2789,11 +2776,8 @@ def _isStandardUniqueName(self, uniqueName: str) -> bool:
         >>> rmd._isStandardUniqueName('average duration')
         False
         '''
-        if super()._isStandardUniqueName(uniqueName):
-            return True
-        if uniqueName in self.additionalRichMetadataAttributes:
-            return True
-        return False
+        return (super()._isStandardUniqueName(uniqueName)
```

Comment:
Could you explain why? To me the short boolean expression would seem much easier to parse/understand quickly.

Reply:
It's much harder for me to keep track of why we're returning what we're returning and it's really not a speed improvement in the system.   In the old system we test one thing and if it's good, we return True, and then we test another thing, and if it's good, we return True.  Then if we get to the end, we return False.  Very easy to look at the logic.  In the new way, I have to look ahead and see "is there going to be an and clause?  Are we going to add the values and do something with it?  Are the parentheses there for a reason or only because the line is too long?"

It just makes me read code slower, and there's no discernible speedup.